index.bs

```diff
@@ -327,6 +369,39 @@ The {{DeviceMotionEvent/rotationRate}} attribute must return the value it was in
 
 The {{DeviceMotionEvent/interval}} attribute must return the value it was initialized to. When the object is created, this attribute must be initialized to 0.
 
+The static {{DeviceMotionEvent/requestPermission()}} operation, when invoked, must run these steps:
+<ol>
+ <li><p>Let <var>promise</var> be a new promise.
+ 
+ <li><p>If the method call was not <a>triggered by user activation</a>, then reject <var>promise</var> with a
+ <code>NotAllowedError</code> exception and return <var>promise</var>.
+
+ <li>
+  <p>Run these steps <a>in parallel</a>:
+  
+  <ol>
+   <li><p>Let <var>permission</var> be <a>permission</a> for
+   <a>entry settings object</a>'s <a for="environment settings object">origin</a>.
+
+   <li><p>If <var>permission</var> is "<code>default</code>", ask the user whether sharing device motion
+   for the <a>entry settings object</a>'s
```

Comment:
Please don't use entry settings object! Use the current settings object. Cf. https://github.com/whatwg/notifications/issues/86